### DISCUSSION TITLE
[Q] RR shows wrong startup info (version)

### DISCUSSION THREAD

**Discussion by sergey-telpuk** [roadrunner-server/roadrunner#1552] [Q&A] [answered]
Hello, I see the following info from RR

**actual**
```bash
 [INFO] RoadRunner server started; version: local, buildtime: development
 ```
**expected:**
version: local - should be 2023.1.2
buildtime: development - time of built

In 2.11.4 was correct.

**Comment by rustatian** [comment, marked as answer]
Hey @sergey-telpuk 👋🏻 
I downloaded a binary and did some double-checking:
`./rr --version` output is: 
![image](https://user-images.githubusercontent.com/8040338/236615127-fcb5e94a-9eb5-4384-82f4-808a573179c6.png)

RR startup info:
![image](https://user-images.githubusercontent.com/8040338/236615150-0df2ecf2-0bd4-4b5f-8e52-1def71ebb047.png)

It seems that you're using your own local build.

**Reply by sergey-telpuk** [reply]
Yes. I use own local build. Any changes to fix ?

**Reply by rustatian** [reply]
There is no bug to fix 😃. Version should be set during the linking stage of the binary compilation, e.g.: 
```
-X github.com/roadrunner-server/roadrunner/v2023/internal/meta.version=<YOUR_VERSION_HERE>
-X github.com/roadrunner-server/roadrunner/v2023/internal/meta.buildTime=<TIME_HERE>
```
ref: https://github.com/roadrunner-server/roadrunner/blob/master/.github/workflows/release.yml#L89
ref: https://pkg.go.dev/cmd/go

**Comment by sergey-telpuk** [comment]
@rustatian hi, thanks for helping. Yes, It was the wrong link for v2023
![image](https://user-images.githubusercontent.com/4964014/236666439-d66c119c-011c-4e6b-880f-2ef18dd08457.png)
